collections/_release_4_4/entry-general-core-manifold-library-replaces-csg-implementation.md

```diff
@@ -0,0 +1,22 @@
+---
+type: entry
+section: general
+subsection: core
+rank: 1
+importance: 4
+anchor: manifold-library-replaces-CSG-implementation
+title: Manifold library replaces CSG implementation
+blockquote: Reaping the benefits of open source
+text: |
+  Support for Constructive Solid Geometry (CSG) has been in the engine since Godot 3.1 and is a great tool for rapid prototyping.
+
+  Nonetheless, our internal implementation suffered from stability issues and other bugs. This is one of the engine areas that does not have a dedicated maintainer to look after it, so when an open source library with the same functionality was released, we jumped at the opportunity.
```

Comment:
```suggestion
  Nonetheless, our internal implementation suffered from stability issues and other bugs. This is one of the engine areas that does not have a dedicated maintainer to look after it, so when an open-source library with the same functionality was released, we jumped at the opportunity.
```

Reply:
This hyphenated form has been standardized so let's use it indeed